modules/backend/formwidgets/RelationManager.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Backend\FormWidgets;
+
+use ApplicationException;
+use Backend\Classes\FormField;
+use Backend\Classes\FormWidgetBase;
+use Lang;
+
+class RelationManager extends FormWidgetBase
+{
+    /**
+     * @inheritDoc
+     */
+    protected $defaultAlias = 'relationmanager';
+
+    /**
+     * @var bool Disables the ability to add, update, delete or create relations.
+     */
+    protected $readOnly;
+
+    /**
+     * @var string path to controller action to open a record.
+     */
+    protected $recordUrl;
+
+    /**
+     * @var string custom JavaScript code to execute when clicking on a record.
+     */
+    protected $recordOnClick;
+
+    public function init(): void
+    {
+        $this->fillFromConfig([
+            'readOnly',
+            'recordUrl',
+            'recordOnClick',
```

Comment:
Let's also add a `relation` property in case someone wants to have a different field name from the relationship definition name.

Reply:
@LukeTowers do you mean to make this call configurable for its first argument ?
```
return $this->controller->relationRender($this->formField->fieldName, $options);
```